### DISCUSSION TITLE
Python-fontforge failed to add Chinese characters.

### DISCUSSION THREAD

**Discussion by PkuDavidGuan** [fontforge/fontforge#4799] [Q&A]
I failed to create a Chinese truetype font from svg files with Python 2.7. 

### What I did:
```python
import fontforge


font = fontforge.font()
glyph = font.createMappedChar(ord(unicode('、', 'utf-8')))
glyph.importOutlines('demo.svg')
```
### The reported errors:

```shell
# failed in font.createMappedChar
ValueError: Encoding is out of range
```

I did not know how to fix the bug and hope anyone could help me.

**Comment by ctrlcctrlv** [comment]
You want `font.createChar`, not `font.createMappedChar`.
